src/picongpu/include/fields/laserProfiles/laserGaussianBeam.hpp

```diff
@@ -53,9 +53,11 @@ namespace picongpu
             //gaussian beam waist in the nearfield: w_y(y=0) == W0
             const float_64 w_y = W0 * sqrt( 1.0 + ( FOCUS_POS / y_R )*( FOCUS_POS / y_R ) );
```

Comment:
also, it is not necessary to cast `W0`. this line does a float calculation and casts the result to float_64 for the following operations.